bson/src/main/org/bson/BsonObjectId.java

```diff
@@ -28,6 +28,13 @@
     private final ObjectId value;
 
     /**
+    * Construct a new instance with a new {@code ObjectId}.
```

Comment:
I think its friendly, I'd like to update the other places where we make the user double construct to do it - but will do it aside from this PR, if its agreed upon :)